beacon_node/beacon_chain/benches/benches.rs

```diff
@@ -35,10 +35,11 @@ fn all_benches(c: &mut Criterion) {
     type E = MainnetEthSpec;
     let spec = Arc::new(E::default_spec());
 
-    let trusted_setup: TrustedSetup = serde_json::from_reader(TRUSTED_SETUP_BYTES)
-        .map_err(|e| format!("Unable to read trusted setup file: {}", e))
-        .expect("should have trusted setup");
-    let kzg = Arc::new(Kzg::new_from_trusted_setup(trusted_setup).expect("should create kzg"));
+    let kzg = if spec.deneb_fork_epoch.is_some() {
+        KZG.clone()
+    } else {
+        KZG_NO_PRECOMP.clone()
+    };
```

Comment:
maybe rather than duplicating this logic, which will need to be tweaked when we enable Peer DAS in more tests, we could define a function in `test_utils` that takes the `ChainSpec` and returns an `Arc<Kzg>` based on the values of `deneb_fork_epoch` and `eip7594_fork_epoch`?